### PR TITLE
[VIS] reordered the achievements and removed unused import

### PR DESCRIPTION
**Pull request by olly-larkin**
# Summary

- Modified order of achievements per @ahm116's suggestion
- Cursor now changes to clickable while hovering over achievement

## Additional Information

NA

### CODE REVIEW

File: website/src/components/Visualisations/Achievements/AcheivementEntries.ts [hunk 1]
```diff
@@ -153,11 +153,17 @@ const turnsInPowerMetricCollection = (data: OutputJSONType) =>
 
 const acheivementList: AcheivementEntry[] = [
```
Comment (Darrekt) [inline website/src/components/Visualisations/Achievements/AcheivementEntries.ts:154]: AchievementEntry

**Review by Darrekt — APPROVED** [review]
Love the functions as data

**Review by augusta100 — APPROVED** [review]
Looks good to me

File: website/src/components/Visualisations/Achievements/AcheivementEntries.ts (hunk 1 above)
Comment (olly-larkin) [reply website/src/components/Visualisations/Achievements/AcheivementEntries.ts:154]: oof

Comment (olly-larkin) [reply website/src/components/Visualisations/Achievements/AcheivementEntries.ts:154]: should this be changed in another PR or leave it?

**Pull request merged**